website_multi_theme/README.rst

```diff
@@ -114,24 +114,16 @@ Known issues / Roadmap
 
 * Private themes support is not guaranteed.
 * There is no UI to remove websites. Do it through an odoo shell.
-* The module includes workarounds that should be removed once merged upstream:
-  * https://github.com/odoo/odoo/pull/17635
-  * https://github.com/odoo/odoo/pull/17970
 * Theme picker should include some kind of thumbnail if possible.
 * If you install any of the supported themes after installing this addon, you
   will have to press *Reload* in the website config wizard to make it notice
   the change.
 * If you install any unsupported theme along with this addon, it would possibly
   become the base for all those supported, which can easily lead to weird
   situations and errors.
-
-Required bug patch
-------------------
-
-You need to include this code in your Odoo instance for the addon to work.
-No workaround possible for this one:
-
-* https://github.com/odoo/odoo/pull/18462
+* This addon will not work if your Odoo is not patched. Make sure it is updated
+  before installing. It should include this commit:
+  https://github.com/odoo/odoo/commit/48fe0a595308722a26afd5361432f24c610b4ba0
```

Comment:
No need, because if you have the specified commit, the other 3 needed commits will be there.